dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/cslb/CslbExtractionPipeline.scala

```diff
@@ -30,14 +35,38 @@ object CslbExtractionPipeline extends ScioApp[Args] {
         .getOrElse(List())
 
   val subdir = "cslb"
-  val arm = List("annual_2020_arm_1")
+
+  // get list of individual dates, then get the set of years and return a list of distinct years
+  def getYearList(start: OffsetDateTime, end: OffsetDateTime): List[Int] = {
+    if (start.isAfter(end)) throw new CslbExtractionFailException
+    val dateList =
+      (Iterator.iterate(start)(_ plusDays 1) takeWhile (_ isBefore end.plusDays(1))).toList
+    dateList.map(date => date.getYear).distinct
```

Comment:
I was originally using "toSet.toList" but switched to using "distinct" because by default, the "Set" interface doesn't preserve element order - caught that from one of the unit tests